roles/dns/README.md

```diff
@@ -1,52 +1,145 @@
-## The rhc-ose ansible dns role
```

Comment:
This update seems potentially picked up from the `dns` feature branch. If so, it should follow that branch rather than this PR as it doesn't match the content of the actual roles if not.